graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Row, Col } from 'react-bootstrap';
+
+import 'components/content-packs/ContentPackDetails.css';
+
+const ContentPackInstallView = (props) => {
+  const { comment } = props.install;
+  const createdAt = props.install.created_at;
+  const createdBy = props.install.created_by;
+  return (<div>
+    <Row>
+      <Col smOffset={1}>
+        <div>
+          <dl className="deflist">
+            <dt>Comment:</dt> <dd>{comment}</dd>
+            <dt>Installed by:</dt> <dd>{createdBy}&nbsp;</dd>
+            <dt>Installed at:</dt> <dd>{createdAt}&nbsp;</dd>
```

Comment:
Please use the `Timestamp` component to format the timestamp. This allows us to format the timestamp in the users/browser timezone.

See: https://graylog2.github.io/frontend-documentation/#timestamp